source/Sale.h

```diff
@@ -34,6 +34,8 @@ class Sale : public std::set<const Item *> {
 	void Add(const Sale<Item> &other);
 
 	bool Has(const Item *item) const;
+
+	std::string description;
```

Comment:
Normally we have variables as private members that can be accesed through getters/setters.
I would prefer to have this here too.

Reply:
Can do